src/quemb/molbe/mbe.py

```diff
@@ -230,6 +233,11 @@ def __init__(
             self.initialize(mf._eri, compute_hf)
         else:
             self.initialize(None, compute_hf, restart=True)
+        if settings.PRINT_LEVEL >= 10:
+            print(
+                f"Time to initialize BE object: {time() - init_start_time}",
+                flush=True,
```

Comment:
IMHO we overdo the flushing.

In this case I would just print. (also applies to the rest)